hsc/crawler.py

```diff
@@ -40,11 +41,13 @@ def __init__(self):
 		self.options = {}
 
 	def login(self, username, password):
-		resp = self.session.get(self.login_url, auth=(username, password))
-		self.cookies = self.session.cookies.get_dict()
-		self.headers = resp.request.headers
-		self.get_number_of_submissions()
-		return self.total_submissions != 0
+		resp = self.session.post(self.login_url, auth=(username, password), headers={'user-agent': self.user_agent})
+		data = resp.json()
+		if data['status']:
+			self.cookies = self.session.cookies.get_dict()
+			self.headers = resp.request.headers
```

Comment:
@rishabhsingh971 Do you know the reason for setting cookies and headers here?

Reply:
Yes, It's for authentication. We set the response cookies and headers sent by server when successfully logged in and then send them in further requests (as done by browsers)